webapp/src/main/webapp/templates/freemarker/body/siteAdmin/siteAdmin-siteStyle.ftl

```diff
@@ -0,0 +1,130 @@
+<div>
+    <h2>Site Styles</h2>
+    <div class="button-group">
+        <a class="button blue" download href="${urls.base}/css/vitro.css">vitro.css</a>
+        <a class="button blue" download href="${urls.base}/css/login.css">login.css</a>
+        <a class="button blue" download href="${urls.base}/css/edit.css">edit.css</a>
+        <a class="button blue" download href="${urls.base}/css/individual/individual-property-groups.css">individual-property-groups.css</a>
+        <a class="button blue" download href="${urls.base}/css/developer/developerPanel.css">developerPanel.css</a>
+        
+        <!-- Samo Wilma i TF-->
+        <a class="button blue" download href="${urls.base}/css/home-page-maps.css">page-createAndLink.css</a> 
+
+
+        <#if themeDir == "themes/wilma">
```

Comment:
Hardcoding theme paths is not an option. It will be broken once theme is renamed for customization.